docs/configuration.md

```diff
@@ -113,3 +113,19 @@ client = genshin.Client(game=genshin.Game.HONKAI)
 
 client.default_game = genshin.Game.HONKAI
 ```
+
+## Proxy Support
+
+Some endpoints may block your IP, you can set up a proxy, aiohttp only support `http/https/ws/wss` proxy for now.
```

Comment:
Considering websockets are never used at any point I do not think there's any need to mention them.

Reply:
Oh, I didn't even know aiohttp supported that. All good then.